fighters/link/src/acmd/smashes.rs

```diff
@@ -106,26 +129,23 @@ unsafe fn attack_lw4(fighter: &mut L2CAgentBase) {
-		ATTACK(fighter, 3, 0, Hash40::new("kneer"), 	12.0, 30, 60, 0, 80, 3.0, 0.0, 0.0, 0.0, None, None, None, 1.0, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.3, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_cutup"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_CUTUP, *ATTACK_REGION_SWORD);
+		ATTACK(fighter, 0, 0, Hash40::new("sword2"), 	14.0, 30, 60, 0, 80, 3.5, 1.5, 0.0, 0.0, None, None, None, 1.2, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.3, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_cutup"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_CUTUP, *ATTACK_REGION_SWORD);
+		ATTACK(fighter, 1, 0, Hash40::new("sword2"), 	14.0, 30, 60, 0, 80, 3.5, 7.5, 0.0, 0.0, None, None, None, 1.2, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.3, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_cutup"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_CUTUP, *ATTACK_REGION_SWORD);
+		ATTACK(fighter, 2, 0, Hash40::new("armr"), 		12.0, 30, 60, 0, 80, 3.0, 1.0, 1.0, 0.0, None, None, None, 1.2, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.3, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_cutup"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_CUTUP, *ATTACK_REGION_SWORD);
+		ATTACK(fighter, 3, 0, Hash40::new("shoulderr"), 12.0, 30, 60, 0, 80, 3.0, 0.0, 0.0, 0.0, None, None, None, 1.2, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.3, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_cutup"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_CUTUP, *ATTACK_REGION_SWORD);
 		AttackModule::set_attack_height_all(boma, app::AttackHeight(*ATTACK_HEIGHT_LOW), false);
 	}
 	wait(lua_state, 2.0);
 	if is_excute(fighter) {
 		AttackModule::clear_all(boma);
 	}  
 	frame(lua_state, 38.0);
-	if is_excute(fighter) {
-		// faf effectively 56 -> 50
-		MotionModule::set_rate(fighter.module_accessor, 1.33);
-	}
+	FT_MOTION_RATE(fighter, 12.0/(56.0-38.0));
 }
 
 pub fn install() {
     install_acmd_scripts!(
+		game_attacks4,
         attack_s4_2,
 		attack_hi4,
 		attack_lw4,
```

Comment:
some more funny indentation. if this doesn't look funny to you, make sure your environment is set up to convert tabs to spaces automatically